src/pages/ImportData.js

```diff
@@ -336,11 +354,22 @@ const ImportDataDestination = ({
 // This method identifies whether an import source is considered protected data;
 // For now this means pfb imports from AnVIL or Biodata Catalyst.
 export const isProtected = (url, filetype) => {
-  const hostname = new URL(url).hostname;
-  const protectedHosts = ['anvil.gi.ucsc.edu', 'anvilproject.org', 'gen3.biodatacatalyst.nhlbi.nih.gov'];
-  return Utils.cond([!filetype || !url, () => false], [filetype.toLowerCase() !== 'pfb', () => false], () =>
-    protectedHosts.some((host) => hostname.endsWith(host))
-  );
+  try {
+    const hostname = new URL(url).hostname;
+    const protectedHosts = [
+      'anvil.gi.ucsc.edu',
```

Comment:
Should this be configged for environment?

Reply:
This works for both `dev.anvil.gi.ucsc.edu` and `prod.anvil.gi.ucsc.edu` and I guess it makes sense to protect the data from anvil no matter what environment we're on?  Wouldn't want to be able to import prod anvil data into dev.

Reply:
I think I asked about this before, but this client side check is for UI enablement only, correct? That is, only the server APIs control what's actually going to work or not, so even if this list is incorrect the main effect is that the UI won't work correctly; either the enablement will be too permissive or too restrictive.

Reply:
Right, good point, what I have here doesn't actually affect what can and can't get imported, it's purely UX.